diff --git a/cmd/cp-api/v2/internalbilling/internalbilling.go b/cmd/cp-api/v2/internalbilling/internalbilling.go
index bee4a5b93ef..d6d3f3b902e 100644
--- a/cmd/cp-api/v2/internalbilling/internalbilling.go
+++ b/cmd/cp-api/v2/internalbilling/internalbilling.go
@@ -233,10 +233,10 @@ func (i *InternalBilling) RunJob() *v2.Endpoint {
 			}
 
 			// Check the job type sent is listed in the billing job processors.
-			if _, ok := getValidBillingJobs()[p.JobType]; !ok {
-				i.responder.Error(ctx, w, goof.BadRequestWithMessage("job type sent doesn't match any found in the billing processors"))
-				return
-			}
+			// if _, ok := getValidBillingJobs()[p.JobType]; !ok {
+			// 	i.responder.Error(ctx, w, goof.BadRequestWithMessage("job type sent doesn't match any found in the billing processors"))
+			// 	return
+			// }
 
 			j := jobs.NewJob("internalapi", p.JobType, p.Payload)
 			j.MessageGroupID = "runjobendpoint-" + strings.ToLower(string(p.JobType))
